src/resolvers/path.cr

```diff
@@ -7,12 +7,13 @@ module Shards
     end
 
     def read_spec(version = nil)
-      spec_path = File.join(local_path, SPEC_FILENAME)
+      spec_path = File.join(Path[local_path].expand(home: true).to_s, SPEC_FILENAME)
+
 
       if File.exists?(spec_path)
         File.read(spec_path)
       else
-        raise Error.new("Missing #{SPEC_FILENAME.inspect} for #{dependency.name.inspect}")
+        raise Error.new("Missing #{SPEC_FILENAME.inspect} for #{dependency.name.inspect}\n  Expected it at #{spec_path}")
```

Comment:
No need for newlines.

Reply:
In fact it's not needed when using `expanded_local_path` that will report that the directory doesn't exist.